src/billing/Project.ts

```diff
@@ -190,8 +240,7 @@ const ProjectDetail = ({
             ]),
           ]),
         ]),
-      _.size(projectUsers) > 1 &&
-        _.size(projectOwners) === 1 &&
+      oneOwnerForSharedWorkspace &&
```

Comment:
This was a bug fix-- we weren't showing the message if there was a single owner and the user wasn't the owner. In that case, `projectUsers.length === 1` because projectUsers will only contain owners if the user isn't an owner.